Automattic/woocommerce-payments
# Record in tracks merchants returning from the kyc reminder email

Fixes #3957

#### Changes proposed in this Pull Request

Record in Tracks the visit to the URL for redirecting merchants to the KYC flow.

The parameter `wcpay-connect-redirect`  in the redirect URL will be set in the onboarding reminder email. Merchants. would visit the URL we're tracking here by clicking this link in that email.

#### Testing instructions
* Make sure Tracking is enabled in WC options. That's in `{your site}/wp-admin/admin.php?page=wc-settings&tab=advanced&section=woocommerce_com`.
* Make your site publicly accessible. `npm run tube:start` would do the trick if your're using Jurassic Tube. I don't recall if this was required but mentioning it just in case

Initial:
* Visit `{website url}/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect&wcpay-connect-redirect=initial` 
* Go to Live Tracks and search `wcadmin_wcpay_kyc_reminder_merchant_returned`
* Wait ~5 minutes after visiting the URL (it could take ~10 mins for the event to show up) 
* Confirm the event has `eventprops [ type: initial, follow: 0 ]` in its properties

Follow-ups
* Visit `{website url}/wp-admin/admin.php?page=wc-admin&path=%2Fpayments%2Fconnect&wcpay-connect-redirect={ 1 | 2 | 3 | 4}` 
* Go to Live Tracks and search `wcadmin_wcpay_kyc_reminder_merchant_returned`
* Wait ~5 minutes after visiting the URL (it could take ~10 mins for the event to show up) 
* Confirm the event has `eventprops [ type: follow, follow: 1 | 2 | 3 | 4 ]` in its properties


-------------------


**Post merge**
